### ISSUE TITLE
Bug on adding tracks to playlist with SqueezeCloud search and browse engines

### ISSUE THREAD

**Issue by D10Z** [CDrummond/lms-material#208]
When searching a track in search field of SqueezeCloud plugin, the result list is well displayed.
But when selecting one track of this list, only the first item is added to the playlist.
On navigation (like Following, Playlits or Likes) sometime selected tracks aren't added to the playlist.

Using the plugin on classic squeezebox interface doesn't imply those mis-behaviours.

Tested on last versions of squeezeboxserver, lms-material and squeezecloud - Linux and Windows.

**Comment by CDrummond** [comment]
Yup, I noticed this 9months ago. SqueezeCtrl had the same issue: https://forums.slimdevices.com/showthread.php?96757-Annouce-Squeeze-Control-a-squeezebox-remote-app-is-now-available-for-Android&p=934637&highlight=soundcloud#post934637

I added a work-around, but this broke Tidal.

**Comment by D10Z** [comment]
OK, I didn't search any occurence of this bug on https://forums.slimdevices.com and did not notice this workaround in ChangeLog.
My bad...

**Comment by CDrummond** [comment]
No need to apologise, I only managed to find it as I remembered the issue. I'll see what I can do.
Have you tried other control points - e.g. iPeng, SqueezeCtl, Squeezer??

**Comment by CDrummond** [comment]
I'm closing this. Squeezecloud is broken. Tried with default web UI, and the 'All Songs' gets different results to the list it is currently showing! Tracks don't play, etc.

SqueezeCtrl has the same 'issue' with SqueezeCloud as Material does - which gives me some confidence its not a Material issue.

I tried re-adding the workaround, but applying only for SqueezeCloud - and whilst it is better, its still broken.
